crates/sargon-uniffi/src/signing/transaction_intent/hd_signature.rs

```diff
@@ -0,0 +1,48 @@
+use crate::prelude::*;
+use sargon::TransactionIntentHash as InternalTransactionIntentHash;
+
+type InternalHdSignatureForTransactionIntent =
+    sargon::HDSignature<InternalTransactionIntentHash>;
+
+#[derive(Clone, PartialEq, Eq, Hash, uniffi::Record)]
+pub struct HdSignatureForTransactionIntent {
```

Comment:
ah, perhaps a small macro to create this HDSignature and the other one. see comment https://github.com/radixdlt/sargon/pull/288/files#r1866008352